frontend/src/store/modules/security.ts

```diff
@@ -1,3 +1,5 @@
+import SsoUserInfoResponse from "@/types/SsoUserInfoResponse";
+
 export default {
```

Comment:
Bitte komplett ausbauen. Dazu einfach in der ZaehlstelleView.vue den Getter aufruf rausnehmen und durch den neuen hasAuthorities ersetzen